server/db/helper.js

```diff
@@ -2,7 +2,7 @@ const {
   Course,
   Concept,
   Answer,
-  User,
```

Comment:
This model needs to be included in the helper file

Reply:
this last PR doesn't satisfy previous comments, it attempts to bandaid a Sequelize error I had, I chained the course.create methods and commented out the idConcept foreign now the tables populate properly with fake testdata need further review before I proceed to fixing ticket #90